src/app/zap-templates/templates/app/attributes/Accessors-src.zapt

```diff
@@ -10,8 +10,8 @@
 #include <app-common/zap-generated/attribute-type.h>
 #include <app-common/zap-generated/ids/Attributes.h>
 #include <app-common/zap-generated/ids/Clusters.h>
-#include <app/util/af.h>
 #include <app/util/attribute-storage-null-handling.h>
+#include <app/util/af.h>
```

Comment:
Why this change?

Reply:
I had removed it and addeed back, this re-ordered. Will move it back.